script/env.ps1

```diff
@@ -0,0 +1,58 @@
+# Install base dependencies from the script/base.sh file
+
+# Install openssl for certificate generation
+choco install -y openssl
+
+# Install playwright tool for integration tests on browsers
+# Ensure npm is added to PATH
+if (-not ([Environment]::GetEnvironmentVariable("Path", [EnvironmentVariableTarget]::Machine) -like "*npm*")) {
+    Write-Host "Adding npm to PATH..."
+    [Environment]::SetEnvironmentVariable("Path", "$env:Path;$(npm prefix -g)\node_modules\.bin", [EnvironmentVariableTarget]::Machine)
+}
+
+npm install -g playwright
+
+# Installing required python packages
+choco install -y python3
+
+# Ensure Python scripts directory is added to PATH
+if (-not ([Environment]::GetEnvironmentVariable("Path", [EnvironmentVariableTarget]::Machine) -like "*Python3*")) {
+    Write-Host "Adding Python scripts to PATH..."
+    [Environment]::SetEnvironmentVariable("Path", "$env:Path;$(Get-Command python3 | Select-Object -ExpandProperty Directory)", [EnvironmentVariableTarget]::Machine)
+}
+
+# Install mkdocs
+choco install -y mkdocs
+
+# Ensure Ruby scripts directory is added to PATH
+if (-not ([Environment]::GetEnvironmentVariable("Path", [EnvironmentVariableTarget]::Machine) -like "*Ruby*")) {
+    Write-Host "Adding Ruby scripts to PATH..."
+    [Environment]::SetEnvironmentVariable("Path", "$env:Path;$(Get-Command ruby | Select-Object -ExpandProperty Directory)", [EnvironmentVariableTarget]::Machine)
+}
+
+# Install ruby for gem command
+choco install -y ruby
+
+# Ensure Gem scripts directory is added to PATH
```

Comment:
Error:

```log
ruby v3.3.0.1 [Approved]
ruby package files install completed. Performing other installation steps.
 The install of ruby was successful.
  Software installed to 'C:\ProgramData\chocolatey\lib\ruby'

Chocolatey installed 2/2 packages.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
Adding Gem scripts to PATH...
Get-Command : The term 'gem' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At C:\Users\dtaas-windows\Downloads\DTaaS\script\env.ps1:39 char:64
+ ... ::SetEnvironmentVariable("Path", "$env:Path;$(Get-Command gem | Selec ...
+                                                   ~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (gem:String) [Get-Command], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.GetCommandCommand

gem : The term 'gem' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the
spelling of the name, or if a path was included, verify that the path is correct and try again.
At C:\Users\dtaas-windows\Downloads\DTaaS\script\env.ps1:43 char:1
+ gem install mdl
+ ~~~
    + CategoryInfo          : ObjectNotFound: (gem:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

pip : The term 'pip' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the
spelling of the name, or if a path was included, verify that the path is correct and try again.
At C:\Users\dtaas-windows\Downloads\DTaaS\script\env.ps1:46 char:1
+ pip install mkdocs-material python-markdown-math mkdocs-open-in-new-t ...
+ ~~~
    + CategoryInfo          : ObjectNotFound: (pip:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

Adding Python scripts to PATH (for pip)...
Select-Object : Property "Directory" cannot be found.
At C:\Users\dtaas-windows\Downloads\DTaaS\script\env.ps1:51 char:86
+ ... Get-Command python3 | Select-Object -ExpandProperty Directory)", [Env ...
+                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (python3.exe:PSObject) [Select-Object], PSArgumentException
    + FullyQualifiedErrorId : ExpandPropertyNotFound,Microsoft.PowerShell.Commands.SelectObjectCommand

```

Reply:
> Error:
> 
> ```
> ruby v3.3.0.1 [Approved]
> ruby package files install completed. Performing other installation steps.
>  The install of ruby was successful.
>   Software installed to 'C:\ProgramData\chocolatey\lib\ruby'
> 
> Chocolatey installed 2/2 packages.
>  See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
> Adding Gem scripts to PATH...
> Get-Command : The term 'gem' is not recognized as the name of a cmdlet, function, script file, or operable program.
> Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
> At C:\Users\dtaas-windows\Downloads\DTaaS\script\env.ps1:39 char:64
> + ... ::SetEnvironmentVariable("Path", "$env:Path;$(Get-Command gem | Selec ...
> +                                                   ~~~~~~~~~~~~~~~
>     + CategoryInfo          : ObjectNotFound: (gem:String) [Get-Command], CommandNotFoundException
>     + FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.GetCommandCommand
> 
> gem : The term 'gem' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the
> spelling of the name, or if a path was included, verify that the path is correct and try again.
> At C:\Users\dtaas-windows\Downloads\DTaaS\script\env.ps1:43 char:1
> + gem install mdl
> + ~~~
>     + CategoryInfo          : ObjectNotFound: (gem:String) [], CommandNotFoundException
>     + FullyQualifiedErrorId : CommandNotFoundException
> 
> pip : The term 'pip' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the
> spelling of the name, or if a path was included, verify that the path is correct and try again.
> At C:\Users\dtaas-windows\Downloads\DTaaS\script\env.ps1:46 char:1
> + pip install mkdocs-material python-markdown-math mkdocs-open-in-new-t ...
> + ~~~
>     + CategoryInfo          : ObjectNotFound: (pip:String) [], CommandNotFoundException
>     + FullyQualifiedErrorId : CommandNotFoundException
> 
> Adding Python scripts to PATH (for pip)...
> Select-Object : Property "Directory" cannot be found.
> At C:\Users\dtaas-windows\Downloads\DTaaS\script\env.ps1:51 char:86
> + ... Get-Command python3 | Select-Object -ExpandProperty Directory)", [Env ...
> +                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>     + CategoryInfo          : InvalidArgument: (python3.exe:PSObject) [Select-Object], PSArgumentException
>     + FullyQualifiedErrorId : ExpandPropertyNotFound,Microsoft.PowerShell.Commands.SelectObjectCommand
> ```

ruby is needed for installing gem, there are some issues with downloading ruby with chocolatey as it doesn't get added to the correct C:\ directory, and hence in the PATH, a better option is to install ruby from the link I've added in env.ps1 above the install ruby chocolatey command, which installs ruby and gem and all the required dependencies and adds it to the PATH